docs/user/tutorials/01-50-10-aws-redis-cluster.md

```diff
@@ -0,0 +1,155 @@
+# Using AwsRedisCluster Custom Resources
```

Comment:
The tutorial is almost identical to [Using AwsRedisInstance Custom Resources](https://github.com/kyma-project/cloud-manager/blob/main/docs/user/tutorials/01-40-10-aws-redis-instance.md). What is the difference for the user? When to use which resource? We should make it explicit in the documentation. Maybe we can describe it in detail in the [Redis](https://github.com/kyma-project/cloud-manager/blob/main/docs/user/00-40-redis.md) document?